pmdarima/preprocessing/exog/fourier.py

```diff
@@ -60,6 +60,35 @@ class FourierFeaturizer(BaseExogFeaturizer, UpdatableMixin):
         which is the default value if not set. The value of ``k`` can be
         selected by minimizing the AIC.
 
+    prefix : str or None, optional (default=None)
+        The feature prefix
+
+    Examples
+    --------
+    >>> import pandas as pd
+    >>> pd.options.display.max_columns = 6  # for display...
+    >>> from pmdarima.preprocessing import FourierFeaturizer
+    >>> from pmdarima.datasets import load_wineind
+    >>> y = load_wineind()
+    >>> trans = FourierFeaturizer(12, 4)
+    >>> y_prime, exog = trans.fit_transform(y)
+    >>> exog.head()
+       FOURIER_0     FOURIER_1    ...      \
```

Comment:
Does the output have that slash, or was that a product of copy/paste?

Reply:
It has that slash

Reply:
Huh, looks funky to me, but as long as it's legit